semantic_conventions/metrics/rpc-metrics.yaml

```diff
@@ -0,0 +1,116 @@
+groups:
+  # TODO: Should we list the attributes on each metric
```

Comment:
All: The YAML now allow us to define the attributes under each metric. That will render the metric table and its attributes. 

Given here all attributes apply to all metrics, should we:

- Be consistent and add all attributes to all metrics. This will duplicate the same attribute tables for each metric. 
- Leave the attributes in its own section like we have today. This reduces duplication a lot with the "downside" of not having the attributes close to the metrics in the page. But given there aren't many metrics, I think it's still OK.

Reply:
I actually think we should do the first BUT for this PR we can keep status quo with the second.  I would NOT close the underling ticket though.